argilla/src/argilla/_api/_http/_client.py

```diff
@@ -27,17 +27,36 @@ class HTTPClientConfig:
     retries: int = 5
 
 
-def create_http_client(api_url: str, api_key: str, **client_args) -> httpx.Client:
+TRANSPORT_ARGS = [
```

Comment:
can't we get thes from the init function signature of the `httpx.HTTPTransport`?

Reply:
@frascuchon If you prefer to use the current implementation, feel free to merge.